app/src/main/java/com/codepath/chefster/client/FirebaseClient.java

```diff
@@ -81,4 +82,10 @@ public void uploadNewDataToDatabase(){
             mDatabase.child("dishes").child(String.valueOf(dish.getUid())).setValue(dish);
         }
     }
+
+    //upload the reviews to database
+    public void uploadDishReviewsToDatabase(Review review, Integer dishId){
```

Comment:
i wrote this method to load the reviews from database, but as we dont have reviews in database, we can comment it now and uncomment it later.